server/logging/CoreLogback.java

```diff
@@ -36,6 +36,8 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static ch.qos.logback.core.CoreConstants.UNBOUNDED_TOTAL_SIZE_CAP;
+import static ch.qos.logback.core.CoreConstants.UNBOUND_HISTORY;
```

Comment:
"ED"?

Reply:
Not our constants, can't do anything about em sadly